ontology/yaml/resources/connections/connections.yaml

```diff
@@ -22,3 +22,5 @@ HAS_PART:
   description: "Source has some component or part defined by Target."
 HAS_RANGE:
   description: "Source has a coverage or detection range defined by Target."
+MEASURES:
+  description: "Source provides data that is representative of Target load."
```

Comment:
This is not a good enough description; need to revise to remove the specificity about "load" -- measures is a more abstract concept where it defines the relationship between a source quantifying an attribute defined by the target